WooCommerce/Classes/POS/Utils/Color+WooCommercePOS.swift

```diff
@@ -90,18 +90,13 @@ extension Color {
     // MARK: - Buttons
 
     static var posPrimaryButtonBackground: Color {
-        return Color(
-            UIColor(
-                light: .withColorStudio(.wooCommercePurple, shade: .shade50),
-                dark: .withColorStudio(.wooCommercePurple, shade: .shade30)
-            )
-        )
+        return Color(.withColorStudio(.wooCommercePurple, shade: .shade40))
```

Comment:
For primary buttons, in the app now, we are using `purple_40` in all themes, this is to give a better contrast with the white text.

But for POS, I noticed the button text is black, was this a design decision or just a dev choice? Personally, I think white looks better and gives a better contrast, if you prefer to keep things as they are, I can update the background to keep using `purple_30` on dark theme for a slightly better contrast than `purple_40`.

| Black | White |
| --- | --- |
| ![Simulator Screenshot - iPad Air 13-inch (M2) - 2024-12-23 at 12 57 01](https://github.com/user-attachments/assets/4ae44cb1-8af7-4134-ac54-de7a18149ce3) | ![Simulator Screenshot - iPad Air 13-inch (M2) - 2024-12-23 at 12 56 17](https://github.com/user-attachments/assets/5e1fda28-62c8-4e8c-96c1-307b566496ee) |